modules/coo-dashboard-ui-plugin-configure.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/dashboard-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-dashboard-ui-plugin-configure-_{context}"]
+= Configuring a dashboard
+
+The dashboard UI plugin searches for datasources as `ConfigMap` resources in the `openshift-config-managed` namespace, that have the label `console.openshift.io/dashboard-datasource: 'true'`. The `ConfigMap` resource must define a datasource type and an in-cluster service where the data can be fetched. 
```

Comment:
May be better
The dashboard UI plugin searches for datasources from `ConfigMap`

Reply:
agreed and done